### PR TITLE
Pramods/home

### PR DESCRIPTION
**Pull request by reddypramod85**
This PR adds a header, footer, home, page layout, routes, arcade page, sticker wall page.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -21,24 +24,31 @@ const App = () => {
   return (
     <Grommet theme={hpe} themeMode="dark">
```
Comment (britt6612) [inline src/App.js:25]:
```suggestion
    <Grommet theme={hpe} full themeMode="dark">
```

Comment (britt6612) [reply src/App.js:25]: You will need this on the Grommet wrapper to fill the parent page

File: src/components/HPEDevFooter/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  //========================================= Footer
+  return (
+    <Header
+      align="center"
+      direction="row"
+      flex={false}
+      justify="between"
+      gap="medium"
+      fill="horizontal"
+      pad="xsmall"
```
Comment (britt6612) [inline src/components/HPEDevFooter/index.js:21]: This pad should be larger the sides are 48px so `horizontal: 'large'` then check with figma for vertical I believe medium should work

**Comment by IanKBovard** [comment]
We should only focus on the layout and routes for this pr. the stickerwall and arcade page should be removed. i see a lot of copy and paste in there that won't work for our app. ill push up a separate branch with some refactoring to show you what i mean.

File: src/components/Footer/index.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import React, { useContext } from 'react';
+import { Box, Text, Footer, ResponsiveContext, Button } from 'grommet';
+
+// import Footer from "./styles";
+export const AppFooter = () => {
+  const size = useContext(ResponsiveContext);
+
+  const footerLinks = [
+    { label: 'Terms' },
```
Comment (britt6612) [inline src/components/Footer/index.js:9]: Im guessing we will need pass `href` to these links..

also these names are not the same as on figma. 

Fix these to be:
Privacy Policy 
Cookies 
Terms of Use
Do Not Sell My Personal Information

**Comment by IanKBovard** [comment]
There are also, a lot of commented out code that should not be merged in.

File: src/components/Header/styles.js [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+  @media (max-width: 960px){
+    img {
+      width: auto;
+      max-width: 98%;
+      max-height: 275px;
+    }
+  }
+`;
+
+export const BrickWall = styled(Box)`
```
Comment (britt6612) [inline src/components/Header/styles.js:22]: Alot of this can be taken. out

**Pull request merged**